exporters/otlp/trace/src/main/java/io/opentelemetry/exporter/otlp/trace/OtlpGrpcSpanExporterBuilder.java

```diff
@@ -123,9 +150,9 @@ public OtlpGrpcSpanExporterBuilder addHeader(String key, String value) {
   public OtlpGrpcSpanExporter build() {
     if (channel == null) {
       final ManagedChannelBuilder<?> managedChannelBuilder =
-          ManagedChannelBuilder.forTarget(endpoint);
+          ManagedChannelBuilder.forTarget(endpoint.getAuthority());
```

Comment:
authority, if I read the spec correctly, also can include auth info. Are we ok with that possibly ending up here, or would be better off explicitly grabbing the host and port and putting them together?

Reply:
either, way, we should probably have a test that uses the full form of the authority section and makes sure we deal with it correctly. :)

Reply:
Good callout - added tests and they actually seem to work. Not sure about authority vs host/port but since it seems to work leaning towards keeping it simple.